command/operator_diagnose.go

```diff
@@ -176,7 +176,12 @@ func (c *OperatorDiagnoseCommand) RunWithParsedFlags() int {
 		c.UI.Output(string(resultsJS))
 	} else {
 		c.UI.Output("\nResults:")
-		results.Write(os.Stdout)
+		w, _, err := term.GetSize(0)
```

Comment:
i'm not sure what's going on here.

i looked up the doc for `term.GetSize()` and it looks like it gives the width, height, and error of the current terminal window. I'm not sure what the `0` input means, or why we'd want to return the width?

Reply:
Yeah, we're trying to determine the with of the terminal we're running in.  The 0 is the file descriptor for "stdout", which identifies the correct TTY for the terminal.  We're only interested in width, which is what we pass to set the wordwrap limit.

Reply:
ohhhhh totally makes sense now. thanks!